components/Mob.js

```diff
@@ -5,12 +5,11 @@ template.innerHTML = `
 
         img {
             display: block;
-            -webkit-user-drag: none;
```

Comment:
This works! but is not the recommended/standard approach, so it will break on other browsers/devices such as Mozilla

Refer to https://caniuse.com/webkit-user-drag